strum_macros/src/macros/strings/display.rs

```diff
@@ -29,7 +29,20 @@ pub fn display_inner(ast: &DeriveInput) -> syn::Result<TokenStream> {
 
         let params = match variant.fields {
             Fields::Unit => quote! {},
-            Fields::Unnamed(..) => quote! { (..) },
+            Fields::Unnamed(ref unnamed_fields) => {
+                // Transform unnamed params '(String, u8)' to '(ref field0, ref field1)'
+                let names: Punctuated<_, Token!(,)> = unnamed_fields
+                    .unnamed
+                    .iter()
+                    .enumerate()
+                    .map(|(index, field)| {
+                        assert!(field.ident.is_none());
+                        let ident = syn::parse_str::<Ident>(format!("field{}", index).as_str()).unwrap();
```

Comment:
I think this use of format!() broke the no_std builds. We see `error: cannot find macro `format` in this scope`. Should this perhaps be `alloc::fmt::format!`?

Reply:
that format is fine as it is run at build time.
The problem is the format in the next chunk which is under a quote!